### DISCUSSION TITLE
[Suggestion] Is the 6th form / Twisted Corner possible with Neovox?

### DISCUSSION THREAD

**Discussion by Kaiser-Indrasil** [tsunamayo/Starship-EVO#4188] [Suggestions]
Hey, Tsuna!

I've been having a blast with the building system in the Experimental branch with the multi-symmetry capability, fixed inter-entity collision checks, and the Neovox scale tech, of course.
I don't need clipping anymore.
Hell, I don't even need to go back to the Legacy branch anymore.
The building system is pretty much fleshed out.

Maybe with the exception of that one last block.
You know, which one.
The one that was unfeasible to introduce because of mirroring issues.
I'm aware that you've heard us those few times we suggested it in the past, but I'm wondering if it's feasible now that the Neovox tech is in? 🤔 

![image](https://user-images.githubusercontent.com/63756057/121607899-c6ff2c80-ca48-11eb-8707-9d1459eb86c5.png)

**Comment by Uncle-Ulty** [comment]
I lost count of how many times this block could help me immensely! As a person who works with several angles and curvatures, every new block is welcome <3

**Comment by elro0003** [comment]
Its the block that's missing from pretty much all building games, and the last block needed to make smooth edge curves for 3d surface curves (currently we can go only up to smooth 2d, with the pyramid blocks), along with many other uses. If it's added, it and the stretching tool would allow for basically any 3d shape to be made, and with the novox it'd be amazing to have it.

**Comment by AsageSoftworks** [comment]
Please Tsuna! This has helped immensely when modded into games like Space Engineers. It allows for the smoothest of complex angle transitions! Please add!

**Comment by ExodistSKY1** [comment]
I would really like to see this block implemented again, I would be okay with it not being able to be mirrored

**Comment by cloudrambler** [comment]
There are some parts of my ship that I literally spent days trying to resolve the inconsistencies between necessary surfaces; mostly without a tidy solution after countess attempts. In most cases those surfaces could be neatly resolved with this final logical form, which would also unlock a lot of creative potential and make the design process significantly more rewarding.

I agree with @ExodistSKY1 that it would be worth including even if certain related functions could not be perfected. In fact as a basic option I would be happy to have the forms (left and right hand) specifically enabled in the debug menu, without mirroring.

**Comment by Servomotor** [comment]
this shape would definitely be handy

**Comment by Dwarf-LordPangolin** [comment]
I'm pretty sure this is the single most-requested shape for the game, with or without mirroring.

**Comment by ProPeach** [comment]
> I'm pretty sure this is the single most-requested shape for the game, with or without mirroring.

Absolultely! It's the only truly essential shape we're missing in my opinion, and would actually cut down on some child entity usage in some cases

**Comment by theppslayer69** [comment]
Yes, i would love this shape, ignore the name i made it as a joke because i wasn't expecting to keep using github

**Comment by Kaiser-Indrasil** [comment]
Just one of many examples I could use this shape in:

![image](https://user-images.githubusercontent.com/63756057/124310790-a0727400-db64-11eb-8096-8777b6b2b977.png)

@tsunamayo What are your thoughts on this block in the context of Neovox?

**Comment by Pawnp** [comment]
![image](https://user-images.githubusercontent.com/72423251/124328480-b2014f00-db57-11eb-9394-59c5d4985218.png)

I would love to have the sixth form in my case of where the red block moves in the direction of the arrow along the blue angled block.

**Comment by elro0003** [comment]
That last picture with 2 intersecting things can already be done with 2 pyramids. But the 6th form would be absolutely amazing either way.

**Comment by Pawnp** [comment]
Sure if the red part was static, but if you read what I wrote the red part moves along the blue angled block.

edit:
updated the picture to stop confusion

**Comment by elro0003** [comment]
Oh, ok. My bad
